CommonMark/CommonMarkException.cs

```diff
@@ -8,7 +8,7 @@ namespace CommonMark
     /// <summary>
     /// An exception that is caught during CommonMark parsing or formatting.
     /// </summary>
-#if v2_0 || v3_5 || v4_0 || v4_5
+#if v2_0 || v3_5 || v4_0 || (v4_5 && !portable)
```

Comment:
Correction: Serialization won't work.

[Possible solution](
http://stackoverflow.com/a/27143002/3583). There has to be a shim for seriallization to work.

Should I add CSShim as a dependency or reimplement its functionality?